.cirrus.yml

```diff
@@ -97,6 +97,9 @@ only_if: $CIRRUS_BRANCH == 'main' || ($CIRRUS_PR != '' && $CIRRUS_BASE_BRANCH ==
 preflight_check_task:
   <<: *IOX2_CONTAINER_UBUNTU_22_04_X64
   set_toolchain_script: rustup default stable
+  fingerprint_script: 
+    - cargo install cargo-audit 
```

Comment:
The installation of cargo-audit needs to go into the docker files in `internal/docker`. At least for Ubuntu. It could also easily be added for Arch Linux but that's not strictly required.

Reply:
... oh, and we might need to add the check to each platform debug build since each platform could use different crates.

Reply:
You're completely right. I've added the installation in the `Dockerfiles`